docs/user-guide/ze-working-with-jobs.md

```diff
@@ -1,27 +1,68 @@
 
-1. Expand **JOBS** in the **Side Bar**.
-2. Open a directory with JCL files.
-3. Right-click on the JCL file you want to view, and select the **Get JCL** option.
+1. Expand the **JOBS** tree in the **Side Bar**.
+2. Click on the **Search** icon next to a profile and enter search criteria.
+
+   Search results display under the profile in the **Side Bar**.
+
+3. Expand a job to view its spool files.
+
+## Viewing job contents
+
+1. Expand the **JOBS** tree in the **Side Bar**.
+2. Click on the **Search** icon next to a profile and enter search criteria.
+
+   Search results display under the profile in the **Side Bar**.
+3. Right-click on the job you want to view, and select the **Get JCL** option.
+
+   Job contents display in an **Editor**.
+
+   :::note
+
+   To edit the job and run it again, make the changes in the **Editor**, right click the **Editor**, and select the **Submit as JCL** option. For instructions, see Step 3 in [Submitting a local file as JCL](../user-guide/ze-working-with-data-sets.md#submitting-a-local-file-as-jcl).
+
+   :::
 
    ![View JOB](../images/ze/ZE-jobs-get-jcl.gif)
 
 ## Downloading spool content
 
-1. Expand **JOBS** in the **Side Bar**.
-2. Open a directory with JCL files.
-3. Click the **Download** icon next to a folder with the spool content.
-4. Save the file on your computer.
+### Downloading spool files from a job
+
+1. Expand the **JOBS** tree in the **Side Bar**.
+2. Click on the **Search** icon next to a profile and enter search criteria.
+
+   Search results display under the profile in the **Side Bar**.
+3. Right-click on the desired job and select either:
+
+   - **Download All** to download all the spool files to a folder on your local disk.
+   - **Download All (Binary)** to download all the spool files in binary formaton your local disk. **[correct?]**
+
+   The selected files are saved in a folder that is the job name in the specified location.
 
    ![Download Spool](../images/ze/ZE-jobs-download-spool.gif)
 
+### Downloading a single spool file
+
+1. Expand the **JOBS** tree in the **Side Bar**.
+2. Click on the **Search** icon next to a profile and enter search criteria.
+
+   Search results display under the profile in the **Side Bar**.
+3. Expand the job containing the desired spool file.
+4. Right-click on the spool file and select either:
+
+   - **Download All** to download all the spool file contents to a folder on your local disk.
+   - **Download All (Binary)** to download all the spool file contents in binary format on your local disk. **[correct?]**
+
+   The selected file is saved in the specified location.
+
 ## Sorting jobs
 
 1. Expand **JOBS** in the **Side Bar**.
 2. Click on the **Sort** icon to the right of a profile.
-3. Select the **Sort Direction** option and select either **Ascending** or **Descending** from the **Quick Pick** field.
+3. In the **Quick Pick**, select the **Sort Direction** option and select either **Ascending** or **Descending**.
```

Comment:
In the Quick Pick > field or something else?

Reply:
it's a dropdown list I believe

Reply:
Leaving it as Quick Pick